src/components/Footer/index.js

```diff
@@ -30,15 +30,19 @@ const showGetInTouch = pathname => {
 
 const getBackgroundColor = pathname => {
   const path = getPathnameWithSlashes(pathname)
-  /**
-   * These pages all have a final white component so we make sure
-   * to add a grey background to give them some distinction
-   */
-  const colors = {
-    greyBg: ['engineering', 'design', 'training ', 'about-us', 'open-source']
-  }
 
-  return Object.keys(colors).find(key => colors[key].includes(path))
```

Comment:
Why has this been changed so drastically? It's turned it into a binary 'greyBg' or 'white' result, whereas previously it should be extended to include any colour...

If you simply added 'case-study' to the `greyBg` value in the `colors` object you'd have achieved the same thing.

Reply:
It actually didn't work. I tried to pass 'case-study' in the array.

Reply:
Are we sure that this was working in the first place? I was surprised that passing 'case-study' in the array didn't work.

Reply:
ok my bad, but because the slug isn't just `case-study`. You can switch `colors[key].includes(path)` to `path.includes(colour[key])` and it  should work